net/portmapper/portmapper.go

```diff
@@ -671,7 +690,7 @@ func (c *Client) Probe(ctx context.Context) (res ProbeResult, err error) {
 		}
 		port := addr.(*net.UDPAddr).Port
 		switch port {
-		case upnpPort:
```

Comment:
Instead of upcasting these ports, can we cast the addr.Port to uint16?

Reply:
Done.